static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -9,26 +9,32 @@ import {
 import { commercialFeatures } from 'common/modules/commercial/commercial-features';
 import { imrWorldwide } from 'commercial/modules/third-party-tags/imr-worldwide';
 import { imrWorldwideLegacy } from 'commercial/modules/third-party-tags/imr-worldwide-legacy';
-import { ias, permutive, twitter, lotame, fbPixel, remarketing, inizio } from '@guardian/commercial-core';
+import {
+    ias,
+    permutive,
+    twitter,
+    lotame,
+    fbPixel,
+    remarketing,
+    inizio,
+} from '@guardian/commercial-core';
 import config from 'lib/config';
-import { isInAuOrNz, isInUsOrCa } from "common/modules/commercial/geo-utils";
+import { isInAuOrNz, isInUsOrCa } from 'common/modules/commercial/geo-utils';
 
 const addScripts = (tags: Array<ThirdPartyTag>): void => {
     const ref = document.scripts[0];
     const frag = document.createDocumentFragment();
     let hasScriptsToInsert = false;
 
     tags.forEach(tag => {
-        if (tag.loaded === true) {
-            return;
-        }
-        if (tag.beforeLoad) {
-            tag.beforeLoad();
-        }
+        if (tag.loaded === true) return;
+
+        if (tag.beforeLoad) tag.beforeLoad();
+
+        // Tag is either an image, a snippet or a script.
         if (tag.useImage === true && typeof tag.url !== 'undefined') {
             new Image().src = tag.url;
-        }
-        if (tag.insertSnippet) {
+        } else if (tag.insertSnippet) {
```

Comment:
Going from `else` &rarr; `else if` is the only real change. Everything else is only aesthetics.